packages/esm-patient-conditions-app/src/conditions/conditions-form.component.tsx

```diff
@@ -1,8 +1,15 @@
-import React, { SyntheticEvent } from 'react';
+import React, { SyntheticEvent, useEffect, useState } from 'react';
```

Comment:
Inconsistent. We partially use `React.useState` and now `useState`. Can we just use either one? Personally, I'd like the `React.useState`. One directly sees that this is a core hook.